installer_scripts/install-yb-voyager

```diff
@@ -11,11 +11,11 @@ ARGS_LINUX=$@
 VERSION="latest"
 
 # This needs to be changed to the latest debezium release tag
-VOYAGER_RELEASE_TAG=${VOYAGER_RELEASE_TAG:-"v1.6.5"}
-DEBEZIUM_VERSION=${DEBEZIUM_VERSION:-"2.3.3-1.6.5"}
+VOYAGER_RELEASE_TAG=${VOYAGER_RELEASE_TAG:-"v1.7.0-rc1"}
+DEBEZIUM_VERSION=${DEBEZIUM_VERSION:-"2.3.3-1.7.0-rc1"}
 
-# the hash corresponds to the yb-voyager/v1.6.5 tag
-YB_VOYAGER_GIT_HASH="05e9d3db3dd3c8dfda1f4167ac552c736563872f" 
+# the hash corresponds to the yb-voyager/v1.7.0-rc1 tag
+YB_VOYAGER_GIT_HASH="02213557c948480fc290b9379bb0ed3fbdbc9c8cdc6b13c0da53a17facd5a371" 
```

Comment:
which version is this ? latest commit on `1.7-dev` - e0d47e87b1550a4bfa9c31446737167fa5fa7e70

Reply:
Fixed